ingest.go

```diff
@@ -1162,7 +1162,7 @@ func (d *DB) Ingest(paths []string) error {
 	if d.opts.ReadOnly {
 		return ErrReadOnly
 	}
-	_, err := d.ingest(paths, ingestTargetLevel, nil /* shared */, KeyRange{}, nil /* external */)
+	_, err := d.ingest(paths, ingestTargetLevel, nil, KeyRange{}, false, nil)
```

Comment:
nit: let's not remove the `/* shared */` stuff

Reply:
artifact of goland refactor, caused more harm than good in this case 😅